src/braket/circuits/braket_program_context.py

```diff
@@ -139,3 +141,11 @@ def add_result(self, result: Results) -> None:
             result (Results): The result object representing the measurement results
         """
         self._circuit.add_result_type(braket_result_to_result_type(result))
+
+    @singledispatchmethod
+    def handle_parameter_value(self, value):
+        return value
+
+    @handle_parameter_value.register
+    def _(self, value: Expr):
+        return FreeParameterExpression(str(value))
```

Comment:
Use a private method for single dispatch; otherwise, the docs start looking really ugly

Reply:
Good call

Reply:
```suggestion
    def handle_parameter_value(self, value):
        return self._handle_parameter_value(value)

    @singledispatchmethod
    def _handle_parameter_value(self, value):
        return value

    @_handle_parameter_value.register
    def _(self, value: Expr):
        return FreeParameterExpression(str(value))
```